web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -951,6 +951,56 @@ span.upsell-price .tax {
   min-height: 0;
 }
 
+.front-page-luova-block {
+  background-image: url("../assets/LuovaBanner.png");
```

Comment:
Going to introduce you to the idea of "mobile first". Because of the way css and asset loading works, you actually want to make the smallest image the default one and then load in the bigger images as the screen gets bigger.

Its not a big change, it mostly involves switching you @media(max-widths: $md) etc to use min-widths and including the bigger images.